boards/remote/include/board.h

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ * Copyright (C) 2015 Zolertia SL
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    board_remote Re-Mote
+ * @ingroup     boards
+ * @brief       Support for the Re-Mote board
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the Re-Mote board
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *              Antonio Lignan <alinan@zolertia.com>
+ */
+
+#ifndef __BOARD_H
+#define __BOARD_H
+
+#include "cpu.h"
+#include "periph/gpio.h"
+
+#ifdef __cplusplus
+ extern "C" {
+#endif
+
+/**
+ * @name The nominal CPU core clock in this board
+ */
+#define F_CPU               (32000000UL)
+
+/**
+ * @name Assign the peripheral timer to be used as hardware timer
+ */
+#define HW_TIMER            TIMER_0
+
+/**
+ * @name Assign the UART interface to be used for stdio
+ * @{
+ */
+#define STDIO               UART_0
+#define STDIO_BAUDRATE      (115200U)
+#define STDIO_RX_BUFSIZE    (64U)
+/** @} */
+
+/**
+ * @name LED pin definitions
+ * @{
+ */
+#define LED_PORT_C          GPIO_C
+#define LED_PORT_D          GPIO_D
+#define LED_RED_PIN         (2)
+#define LED_GREEN_PIN       (5)
+#define LED_BLUE_PIN        (3)
+
```

Comment:
here (and in various other places in this file) are some unnecessary newlines you could remove.